### DISCUSSION TITLE
please star

### DISCUSSION THREAD

**Discussion by sulincix** [Schweinepriester/github-profile-achievements#604] [Q&A]
https://github.com/turkman-linux/ymp

**Comment by ArielMAJ** [comment]
@sulincix I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@sulincix I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again!

**Reply by sakshgupta** [reply]
hey i can start you repo if u can do mine
[https://github.com/sakshgupta/Local_Seva](https://github.com/sakshgupta/Local_Seva)

**Reply by PARKER-X** [reply]
> hey i can star you repo if u can do mine https://github.com/sakshgupta/Local_Seva

Bro can u star me I star ur repo

**Reply by sakshgupta** [reply]
> > hey i can start you repo if u can do mine https://github.com/sakshgupta/Local_Seva
> 
> Bro can u star me I start ur repo

sure i can star yours give me link to your repo and plz do mine

**Reply by PARKER-X** [reply]
> > > hey i can start you repo if u can do mine https://github.com/sakshgupta/Local_Seva
> > 
> > 
> > Bro can u star me I start ur repo
> 
> sure i can star yours give me link to your repo and plz do mine

https://github.com/PARKER-X/BarBar

**Reply by sakshgupta** [reply]
> > > > hey i can start you repo if u can do mine https://github.com/sakshgupta/Local_Seva
> > > 
> > > 
> > > Bro can u star me I start ur repo
> > 
> > 
> > sure i can star yours give me link to your repo and plz do mine
> 
> https://github.com/PARKER-X/BarBar

done bro
https://github.com/sakshgupta/Local_Seva

**Comment by itspasindu** [comment]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by sakshgupta** [reply]
done bro can u do mine plz
[https://github.com/sakshgupta/Local_Seva](https://github.com/sakshgupta/Local_Seva)

**Comment by PARKER-X** [comment]
@sulincix
Done bro
 https://github.com/PARKER-X/BarBar
https://github.com/PARKER-X/carpido/

**Comment by itspasindu** [comment]
> ##
> * [    
>         Oldest
>       
>   ](/Schweinepriester/github-profile-achievements/discussions/604?sort=old)
> * [    
>         Newest
>       
>   ](/Schweinepriester/github-profile-achievements/discussions/604?sort=new)
> * [    
>         Top
>       
>   ](/Schweinepriester/github-profile-achievements/discussions/604?sort=top)

Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by sakshgupta** [reply]
done can you do mine https://github.com/sakshgupta/Local_Seva

**Comment by MohamedSelimMaazouz993** [comment]


**Reply by ArielMAJ** [reply]
@MohamedSelimMaazouz993 I starred you, ty for the star too!

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Reply by PARKER-X** [reply]
https://github.com/PARKER-X/BarBar done bro star my repo

**Reply by sakshgupta** [reply]
done bro can you do mine too https://github.com/sakshgupta/Local_Seva
ty in advance

**Comment by om3ga6400** [comment]
@sulincix https://github.com/OMEGA6405/OMEGA6405

**Reply by sakshgupta** [reply]
https://github.com/sakshgupta/Local_Seva done can you do mine

**Comment by DevElkami** [comment]
Hi everyone,

[Password Vault](https://github.com/DevElkami/PasswordVault) is a personal password vault with firefox password importer. This vault use Advanced Encryption Standard (AES) encryption (MAUI and Winform).

I would appreciate it if you could check it out and give it a star. Thank you.

**Reply by sakshgupta** [reply]
done can you do mine https://github.com/sakshgupta/Local_Seva

**Reply by ArielMAJ** [reply]
@sakshgupta I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by sakshgupta** [reply]
> @sakshgupta I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
> Feel free to contribute as well, there are many somewhat "simple" issues.

Ty, i stared yours too
Will try to contribute

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)

**Reply by sakshgupta** [reply]
Can you do mine too https://github.com/sakshgupta/Local_Seva
Drop me urs will do the same

**Reply by YuZhangWang** [reply]
> Can you do mine too [sakshgupta/Local_Seva](https://github.com/sakshgupta/Local_Seva) Drop me urs will do the same

I'm happy to do it. Can I work with you on some achievements?
I need to complete the achievements of Pair Extraordinaire, Starstruck and Galaxy Brain, can we use email to discuss how to complete them?

Here are some specific practices.

For Starstruck:[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)
For Galaxy Brain:In any of your repositories, turn on the discussion function, then you propose two discussions for me to answer, and mark my answers as answers.
For Pair Extraordinaire：Coauthored with me in a merged pull request.

**Reply by sakshgupta** [reply]
Sure we can work together.
As I have completed the Pair Extraordinaire, I can help you with that but for the rest I also need them so will need your help too
You can contact me on [sakshamgupta.dev@gmail.com](mailto:sakshamgupta.dev@gmail.com) 
We can continue over there.

**Reply by YuZhangWang** [reply]
> Sure we can work together. As I have completed the Pair Extraordinaire, I can help you with that but for the rest I also need them so will need your help too You can contact me on [sakshamgupta.dev@gmail.com](mailto:sakshamgupta.dev@gmail.com) We can continue over there.

I have sent you an email, please see the latest one.

**Reply by ArielMAJ** [reply]
If you wish we can work on simple issues over my repo and do co-authored commits to get your achievement: 
https://github.com/ArielMAJ/test.me/

**Reply by YuZhangWang** [reply]
> If you wish we can work on simple issues over my repo and do co-authored commits to get your achievement: [ArielMAJ/test.me](https://github.com/ArielMAJ/test.me/)

Okay, I will now try to get Pair Extraordinaire.

**Comment by Makentosh** [comment]
❤️
